Solutions/Nasuni/Analytic Rules/RansomwareAttackDetected.yaml

```diff
@@ -0,0 +1,50 @@
+id: 6c8770fb-c854-403e-a64d-0293ba344d5f
+name: Ransomware Attack Detected
+description: 'Identifies ransomware attacks detected by the Ransomware Protection service running on a Nasuni Edge Appliance.'
+kind: Scheduled
+severity: High
+requiredDataConnectors:
+  - connectorID: NasuniEdgeAppliance
+    datatypes: 
+      - Syslog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+status: Available
+tactics:
+  - Impact
+relevantTechniques:
+  - T1486
+query: |-
+  Syslog
```

Comment:
@mdriscoll-nasuni can you help uploading few sample logs? to test these rules and queries?

Reply:
[query_data.csv](https://github.com/Azure/Azure-Sentinel/files/12045020/query_data.csv)

Reply:
thankyou. tested output by running queries